client/layout/universal-navbar-header/nav-style.scss

```diff
@@ -0,0 +1,767 @@
+$studio-blue-15: #e5f4ff;
+$studio-blue-30: #0675c4;
+
+.x-hidden {
+	border: 0;
+	clip: rect(0 0 0 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+}
+
+.is-section-plugins {
+	[data-calypso-ssr="true"] {
```

Comment:
This might not be the best option because not all logged-out Plugins pages are or will be SSR rendered.
ATM the Categories and details pages aren't SSR and these styles won't apply.

Because of inconsistent user bootstrapping on development environments, you also have SSR for logged-in users on `/plugins` (ie. without a site selected)

Another approach would be to target `.is-section-plugins.has-no-sidebar` as this should be the class added to logged-out layout.